### DISCUSSION TITLE
[Auth] Automatic Vault Backups to User-Specified Location on phone

### DISCUSSION THREAD

**Discussion by Jishnuraj9** [ente-io/ente#1921] [Enhancements]
Request for a feature that enables Auth to automatically back up the codes to a user-specified location without requiring user intervention.

**Comment by andbenn** [comment]
Aegis has very nice functionality in this area.  This could probably be phased in with Ente Auth.

* Integration with Android backup
* Save a backup (w/encryption) to a different area on the file system [which then might be sync'd to a cloud storage]
* Number of backups to keep in that different area
* If you don't do a backup every couple of days, and have new activity, I think it alerts you (double check me on this) to make a backup

**Reply by kaizokan** [reply]
> If you don't do a backup every couple of days, and have new activity, I think it alerts you (double check me on this) to make a backup

correct:
- Show a reminder to backup your vault in case you have not backed up your latest changes
- Makes automatic backups on external storage when changes are made. Only for encrypted vaults. 

All of the backup option can be enabled or disabled.

**Comment by richard7ac** [comment]
Just onboarding from Aegis and looking for this same feature. Online sync is great but having a reliable local backup is just as important. Automating this helps to make sure it is reliable.

**Comment by ua741** [comment]
Similar request: https://github.com/ente-io/ente/discussions/2390
